### PR TITLE
fix(landing): caps + btn alignment

### LINKED ISSUE #8277: Landing Page Bug Fixes Round 2
#### Fixes

1. Let's align the buttons here:

Screen Shot 2021-01-20 at 15 26 43

2. Let's keep the capitalization consistent: earn -> Earn

3. Add a comma here -> ("learning new technologies and..." -> "learning new technologies, and ...")

Screen Shot 2021-01-20 at 15 26 07

4. can we also explore the possibility of bringing the buttons up so that we can immediately see them when we land on the landing page?

Screen Shot 2021-01-26 at 11 53 37

5. Add a small login (like what we have at the right top hand side of gitcoin.co/grants when one logs out) to the landing page, which leads to our github authentication flow

Screen Shot 2021-01-27 at 12 28 58

@PixelantDesign please backfill any other changes you see on this ticket.

### PR DESCRIPTION
**Pull request by chibie**
##### Description

landing page fixes

##### Refers/Fixes

#8277 

##### Testing
https://www.loom.com/share/c3462421f6c547ebb5e61908272466ee

### CODE REVIEW

**Comment by aamustapha** [comment]
Please add screenshot

**Comment by aamustapha** [comment]
Also, based on the codes and the description, I can't see the bit that does  "btn alignment".

Did you miss some commits?

**Comment by chibie** [comment]
@aamustapha i've added the missing commit  👀

**Comment by aamustapha** [comment]
Is there a specific reason you're adding container? Also, can you add a fullscreen screenshot so that we see as it aligns with the rest of the content.

**Comment by frankchen07** [comment]
were numbers 3 & 4 also addressed on the ticket? https://github.com/gitcoinco/web/issues/8277

**Comment by chibie** [comment]
@aamustapha here's a full screenshot

image


@frankchen07 

1,2,3 are done.

can you provide more explanation for 4? should the buttons go above the paragraphs?

**Comment by thelostone-mc** [comment]
Is this ggod to review ?

**Comment by chibie** [comment]
@thelostone-mc  yes it is

File: app/retail/templates/home/index2020.html [hunk 1]
```diff
@@ -121,26 +123,40 @@ <h1>
...
-        <h2>Here to earn?</h2>
-        <p>
-          Build exciting projects and upskill on development while earning
-          Ethereum and other tokens. Find mentorship by connecting with
-          talented developers and leaders of Ethereum’s top companies.
-        </p>
-        <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
-        <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
-        <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+    <div class="container">
```
Comment (aamustapha) [inline app/retail/templates/home/index2020.html:126]: Adding this container wrapper won't make this block align with the rest of the page.

File: app/retail/templates/home/index2020.html [hunk 2]
```diff
@@ -121,26 +123,40 @@ <h1>
...
-        <a class="btn fe-pulse" href="/grants"><span>Support Grant Projects</span></a>
+      <div class="row actions-reverse">
+        <div class="col-md-8">
+          <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+          <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
+          <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+        </div>
+        <div class="col-md-4">
+          <a class="btn fe-pulse" href="/grants"><span>Support Grant Projects</span></a>
+        </div>
```
Comment (aamustapha) [inline app/retail/templates/home/index2020.html:159]: This bit is duplicated from L135

**Review by aamustapha — CHANGES_REQUESTED** [review]

**Comment by frankchen07** [comment]
@chibie number 4 I was thinking something like this.

we bring the buttons up so they don't have to scroll to get to the buttons

Screen Shot 2021-01-26 at 11 53 37

this is taking in some of Joe's comments about having our CTAs being above the fold. 

cc @PixelantDesign, as design czar, what do you think?

**Comment by gcc20** [comment]
oh this makes sense now @frankchen07

**Comment by chibie** [comment]
oh this makes sense now @frankchen07

File: app/retail/templates/home/index2020.html (hunk 2 above)
Comment (gcc20) [reply app/retail/templates/home/index2020.html:159]: no it's not duplicated. they are different blocks... notice the class in L151 and L135 @aamustapha

Comment (chibie) [reply app/retail/templates/home/index2020.html:159]: no it's not duplicated. they are different blocks... notice the class in L151 and L135 @aamustapha

File: app/retail/templates/home/index2020.html (hunk 1 above)
Comment (chibie) [reply app/retail/templates/home/index2020.html:126]: did you try running this locally? @aamustapha

Comment (thelostone-mc) [reply app/retail/templates/home/index2020.html:126]: why not just use row? 
Isn't the entire page wrapped within a container already ?

Comment (chibie) [reply app/retail/templates/home/index2020.html:126]: @thelostone-mc no it isn't. i introduced bootstrap... didn't want to mess with the existing classes

Comment (aamustapha-gitcoin) [reply app/retail/templates/home/index2020.html:126]: I don't have to run it. .container would set a max width for the div.

File: app/retail/templates/home/index2020.html (hunk 2 above)
Comment (aamustapha) [reply app/retail/templates/home/index2020.html:159]: Yes. `.container` would set a max width for the div.

**Comment by frankchen07** [comment]
adding one more thing @chibie, apologies for the scope changes.

5. Add a small login (like what we have at the right top hand side of gitcoin.co/grants when one logs out) to the landing page, which leads to our github authentication flow

Screen Shot 2021-01-27 at 12 28 58

This is based off of feedback from existing gitcoin users that they don't have a reliable way to login when they reach gitcoin.co. It eventually takes them to townsquare which isn't the optimal experience but it's a compromise between the landing page CTAs and existing users who just want to login, know the platform, and navigate to where they needed to go.

File: app/retail/templates/home/index2020.html (hunk 1 above)
Comment (thelostone-mc) [reply app/retail/templates/home/index2020.html:126]: argh :/ mannn this is painful 
cool cool makes sense

File: app/assets/landingpage/app.css [hunk 3]
```diff
@@ -638,6 +638,10 @@ section#intro {
     margin-left: 1.5em;
     margin-right: 1.5em; } }
 
+section#whatisgitcoin .actions {
+  display: none
+}
```
Comment (thelostone-mc) [inline app/assets/landingpage/app.css:643]: let's remove this and use `d-none` / `d-md-none` / etc

File: app/assets/landingpage/app.css [hunk 4]
```diff
@@ -659,6 +663,12 @@ section#whatisgitcoin {
       margin-right: 26px; }
 
 @media (max-width: 688px) {
+  section#whatisgitcoin .actions {
```
Comment (thelostone-mc) [inline app/assets/landingpage/app.css:666]: use `d-block`

File: app/assets/landingpage/app.css [hunk 5]
```diff
@@ -659,6 +663,12 @@ section#whatisgitcoin {
...
+    display: block;
+  }
+  section#whatisgitcoin .actions-reverse {
```
Comment (thelostone-mc) [inline app/assets/landingpage/app.css:669]: use d-none

File: app/retail/templates/home/index2020.html [hunk 6]
```diff
@@ -121,26 +123,40 @@ <h1>
...
+          <h2>Here to Earn?</h2>
+          <p>
+            Build exciting projects and upskill on development while earning
+            Ethereum and other tokens. Find mentorship by connecting with
+            talented developers and leaders of Ethereum’s top companies.
+          </p>
+          <div class="actions">
+          <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+          <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
+          <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
```
Comment (thelostone-mc) [inline app/retail/templates/home/index2020.html:138]: thoughts on using the static URL ? (like on the other pages)

File: app/retail/templates/home/index2020.html [hunk 7]
```diff
@@ -121,26 +123,40 @@ <h1>
...
-          ecosystem. Get involved by funding, supporting, or mentoring
-          projects to help push the Ethereum forward.
-        </p>
-        <a class="btn fe-pulse" href="/grants"><span>Support Grant Projects</span></a>
+      <div class="row actions-reverse">
+        <div class="col-md-8">
+          <a class="btn fe-pulse" href="/hackathon-list"><span>Join a Hackathon</span></a>
+          <a class="btn fe-pulse" href="/explorer"><span>Work on a Bounty</span></a>
+          <a class="btn fe-pulse" href="/grants"><span>Get Grant Funding</span></a>
+        </div>
```
Comment (thelostone-mc) [inline app/retail/templates/home/index2020.html:156]: @frankchen07 is this intended ? we show the same actions for 
Here to earn and Here to support ?

**Review by thelostone-mc — CHANGES_REQUESTED** [review]
Left a few comments 
and a question for @frankchen07

**Comment by chibie** [comment]
@PixelantDesign @frankchen07 

image

**Comment by frankchen07** [comment]
@chibie looks like we're missing "the Ethereum [ecosystem] forward." under "Here to Support"

can you add a video too?

**Comment by chibie** [comment]
@frankchen07 i just added a screen recording to description

**Comment by frankchen07** [comment]
hey @chibie thanks for this - just one small thing - is it possible to do the login just like the Grants page, with the same hover over effect for consistency?

like this: https://www.loom.com/share/753abb5b8e9946b188ab47133610d80f

after this, it will be LGTM

We'd like to get this deployed early next week.

**Comment by chibie** [comment]
@frankchen07 

login hover effect ✅ 
redirect to townsquare ✅ 

image

**Review by frankchen07 — APPROVED** [review]
LGTM, but next time please be sure to post a video showing the functionality too! 

Thanks!

**Comment by chibie** [comment]
@frankchen07 here's a vid

https://www.loom.com/share/77a2306fa2e8452e967b99fe1a30c9ba

**Review by thelostone-mc — APPROVED** [review]

File: app/retail/templates/home/index2020.html [hunk 8]
```diff
@@ -13,30 +14,45 @@
...
   <!-- fixed logo-->
   <div id="logo">
     <a href="#">
-      <img src="/static/landingpage/logo_animated.svg" alt="logo" />
+      <img src="{% static "landingpage/logo_animated.svg" %}" alt="logo" />
     </a>
   </div>
+  {% if not user.is_authenticated %}
+  <div class="float-right pt-3 pr-4">
+    <!-- <a class="text-white" href="{% url 'social:begin' 'github' %}?next={{ request.get_full_path }}" onclick="dataLayer.push({'event': 'login'});">
```
Comment (octavioamu) [inline app/retail/templates/home/index2020.html:42]: dead code

**Review by octavioamu — CHANGES_REQUESTED** [review]
just a comment about dead code and Aditya reviews and will be good to merge

File: app/assets/landingpage/app.css (hunk 3 above)
Comment (chibie) [reply app/assets/landingpage/app.css:648]: i tried this... but didn't work as required. @thelostone-mc

**Pull request merged**